### PR TITLE
Add unittest to check JSON resource formats

### PR DESCRIPTION
**Pull request by deathaxe**
This PR proposes to add a unittest to check for malformed JSON resources.

The goal is to ensure valid JSON resources, even though ST might forgive some format errors.

The great test itself was created by @facelessuser. I merged his two files used in mdpopups to get one test_json.py out of it.

-----

The maintainers of this repo require that all pull request submitters agree and adhere to the following:

- [x] I have read and agree to the [contribution guidelines](https://github.com/divmain/GitSavvy/blob/master/CONTRIBUTING.md).
 _(required)_
- [ ] All related documentation has been updated to reflect the changes made. _(required)_
- [x] My commit messages are cleaned up and ready to merge. _(required)_

The maintainers of this repository require you to select the semantic version type that
the changes in this pull request represent.  Please select one of the following:
- [ ]  major
- [ ]  minor
- [x]  patch
- [ ]  documentation only

### CODE REVIEW

File: tests/test_json.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
+"""
+Validate JSON format.
+
+Licensed under MIT
```
Comment (stoivo) [inline tests/test_json.py]: Maybe add a link here
https://github.com/facelessuser/sublime-markdown-popups/tree/master/tests
or 
https://github.com/facelessuser/sublime-markdown-popups/tree/36c0022a5957b288014b33be6212890e332d0cb3/tests

**Comment by stoivo** [comment]
Can you squash the 3. commit into the first one?
Then I will merge

**Comment by deathaxe** [comment]
Done.

**Comment by stoivo** [comment]
Ops, I see that I could have been misunderstood.
What I meant was
Can you squash the 3th commit into the 1st commit, so we will have two commits?

Sorry

Can you fix that? You can use reflog and go to the state before the squash.

**Comment by deathaxe** [comment]
Wait. Fixup! was wrong commit.

**Comment by deathaxe** [comment]
Now it should be ok

**Pull request merged**